.github/workflows/docker.yml

```diff
@@ -1,8 +1,10 @@
 name: Docker
 on:
   push:
+    branches:
+      - 'master'
     tags:
-      - 'v*'
+      - 'v*.*.*'
```

Comment:
Not sure why this change, and not sure if it works actually. The usual way is to use `v*` which would match `v2.11.11` or whatever tag that starts with a `v`.

Reply:
yeah also here, from the template ;)